code/headers/beetle.hpp

```diff
@@ -43,5 +51,12 @@ namespace r2d2::moving_platform {
          * @param    int8_t: new degrees
          */
         void turn(int8_t degrees) override;
+        /**
+         * @brief
+         * This function will move the platform a certain distance in cm
+         * @param    uint16_t: distance
+         * Minimal moving distance is 10 cm.
```

Comment:
Is this meant in the param or in the brief description?

Reply:
param description